instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -1,21 +1,131 @@
 import React from "react";
-import './CommentSection.css';
-import PropTypes from 'prop-types';
+import "./CommentSection.css";
+import PropTypes from "prop-types";
+import Comment from "./Comment";
+import styled from "styled-components";
 
-const CommentSection = props => {
-    return(
-        <div className="comment-container">
-            <p><span>{props.data.username}</span> {props.data.text}</p>
+const AddComment = styled.form`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 2rem 0rem;
+  border-top: 1px solid lightgrey;
+  margin-top: 1rem;
+`;
+
+
+const Input = styled.input`
+width:95%;
+border: none;
+font-size: 2rem;
+`;
+
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: props.data,
+      comments: props.comments,
+      inputText: "",
+      index: props.comments.length
+    };
+  }
+
+  componentDidMount() {
+    const comments = JSON.parse(localStorage.getItem("comments"));
+    if (comments) {
+      this.setState({ comments: comments });
+    }
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here; for example:
```javascript
handleChange = ({ target }) => {
  const { name, value } = target;
  this.setState({ [name]: value });
};
```